src/handlers/staking_handlers.ts

```diff
@@ -2,7 +2,7 @@ import * as express from 'express';
 import * as HttpStatus from 'http-status-codes';
 
 import { StakingDataService } from '../services/staking_data_service';
-import { StakingPoolsResponse } from '../types';
+import { DelegatorResponse, StakingPoolsResponse } from '../types';
```

Comment:
One nit: Can we namespace all our response objects with `StakingXXXResponse`. A lot of Responses will be in that types file in the future...